doc/torque_api.md

```diff
@@ -0,0 +1,167 @@
+# Torque API Methods
```

Comment:
It's all just in one place. You might be better able to better visualize this by previewing the staging link (actual end-user output).